src/Avalonia.Diagnostics/Diagnostics/ViewModels/ResourceSetterViewModel.cs

```diff
@@ -8,20 +8,22 @@ internal class ResourceSetterViewModel : SetterViewModel
 
         public IBrush Tint { get; }
 
-        public ResourceSetterViewModel(AvaloniaProperty property, object resourceKey, object resourceValue, bool isDynamic) : base(property, resourceValue)
+        public ResourceSetterViewModel(AvaloniaProperty property, object resourceKey, object? resourceValue, bool isDynamic) : base(property, resourceValue)
         {
             Key = resourceKey;
             Tint = isDynamic ? Brushes.Orange : Brushes.Brown;
         }
 
         public void CopyResourceKey()
         {
-            if (Key is null)
+            var textToCopy = Key?.ToString();
```

Comment:
I guess this is not need anymore since resource key has been checked when constructing this vm.

Reply:
Yes, it seems to be my copy paste from another place. Not a big issue though.